ucm2/conf.d/macaudio/iMac J456.conf

```diff
@@ -5,5 +5,10 @@ SectionUseCase."HiFi" {
 	# the same config. (That is until there will be support for the
 	# internal mic on the laptops.)
 	Comment "Default"
-	File "laptops1/HiFi.conf"
+	Define.JackCodec "cs42l83"
+	Define.PlaybackChannels "4"
```

Comment:
we have no speakers in out dts yet but imacs have 6 speakers with force cancelling woofers